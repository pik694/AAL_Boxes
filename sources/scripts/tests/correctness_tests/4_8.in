4
79 64 87
48 18 78
83 42 3
12 71 79

