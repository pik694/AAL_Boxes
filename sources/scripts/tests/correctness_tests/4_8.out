2 3 1 0 
585030->517638
